.github/workflows/playwright.yaml

```
@@ -67,10 +67,12 @@ jobs:
         with:
           path: 'galaxy root/.venv'
           key: gxy-venv-${{ runner.os }}-${{ steps.full-python-version.outputs.version }}-${{ hashFiles('galaxy root/requirements.txt') }}-playwright
+      - name: Install dependencies
+        run: ./scripts/common_startup.sh --dev-wheels --skip-client-build
+        working-directory: 'galaxy root'
       - name: Install Playwright
         run: |
-          pip install 'playwright==1.55.0'
-          playwright install chromium --with-deps
+          . .venv/bin/activate && playwright install chromium --with-deps
```

```suggestion
          . .venv/bin/activate
          playwright install chromium --with-deps
```

We are doing exactly the same in `.github/workflows/toolshed.yaml`